tasking/pulp/tasking/constants.py

```diff
@@ -4,6 +4,20 @@
     # The interval in seconds for which the Celery Process monitor thread sleeps between
     # checking for missing Celery processes.
     CELERY_CHECK_INTERVAL=60,
+    # The interval in seconds during which the Celery Processes being monitored need to report a
+    # hearbeat to be considered active.
+    CELERY_TIMEOUT_SECONDS=200,
+    # The interval in seconds with which a secondary resource manager tries to acquire the lock
+    # that is needed to be the primary resource manager.
+    CELERY_TICK_DEFAULT_WAIT_TIME=90,
     # Resource manager worker name
-    RESOURCE_MANAGER_WORKER_NAME='resource_manager'
+    RESOURCE_MANAGER_WORKER_NAME='resource_manager',
+    # celerybeat constants
+    SCHEDULER_WORKER_NAME = 'pulp_celerybeat',
+    # The maximum amount of time in seconds celerybeat sleeps before checking for more scheduled
+    # work to dispatch.
+    CELERY_MAX_INTERVAL=90,
+    # The amount of time in seconds that a celerybeat lock is valid for. If the lock is older than
+    # the value specified here, another instance of pulp_celerybeat is allowed to acquire the lock.
```

Comment:
s/another instance of pulp_celerybeat is allowed to acquire the lock/the process holding the lock is considered to be dead/